### PR TITLE
Tweak wording

### PR DESCRIPTION
**Pull request by iHiD**
What are the rules about how many words per line etc? Is this a python or discord convention?

### CODE REVIEW

**Review by kytrinyx — APPROVED** [review]
I can't speak to the line length thing, but the text looks good to me.

**Comment by iHiD** [comment]
@issacg Could we also link to https://exercism.org/docs/community/being-a-good-community-member/the-words-that-we-use pls.

Maybe:
> At Exercism, we try to ensure that the community is actively welcoming to people of all backgrounds and genders, and this is a small thing that you can do to help. **You can learn more here**.

(Where "learn more here" is a link)

**Comment by IsaacG** [comment]
> What are the rules about how many words per line etc? Is this a python or discord convention?

https://github.com/exercism/discord-bot-py/blob/main/.flake8

It's arbitrarily set to 100 at the moment. Maybe I'll try to figure out how to add a CI test to run all the linters.

**Comment by IsaacG** [comment]
> @issacg Could we also link to https://exercism.org/docs/community/being-a-good-community-member/the-words-that-we-use pls.
> 
> Maybe:
> 
> > At Exercism, we try to ensure that the community is actively welcoming to people of all backgrounds and genders, and this is a small thing that you can do to help. **You can learn more here**.
> 
> (Where "learn more here" is a link)

Discord markdown doesn't support embedded links. You can add a link, but it has to be in plaintext.

**Review by IsaacG — APPROVED** [review]

**Comment by IsaacG** [comment]
@iHiD How's this look?

File: cogs/inclusive_language.py [hunk 1]
```diff
@@ -13,12 +13,15 @@
 logger = logging.getLogger(__name__)
 
 MESSAGE = (
-    "Hello 👋 This is a friendly request that you use gender-neutral nouns when greeting people. "
-    'Rather than saying "hey guys" or "hey dudes", use something like "hey everyone", '
-    'or "hey folks". '
+    "Hello 👋 This is a friendly (automated) request that you don't use gendered pronouns "
```
Comment (iHiD) [inline cogs/inclusive_language.py:16]:
```suggestion
    "Hello 👋 This is a friendly (automated) request to not use gendered pronouns "
```

**Comment by iHiD** [comment]
Yep. Happy to merge!

**Pull request merged**